data/master.tsv

```diff
@@ -1743,6 +1747,8 @@ Time:wait.V	Motion:wait (for).V	To refrain from acting or moving for some durati
 Time:winter.N	Time:winter.N	Traditionally, the fourth of the seasons, marked by the applicable hemisphere of the planet being at its minimum angle of exposure to the Sun, resulting in short days, long nights and typically in low temperatures.	1471	14-74	Winter::N	3273
 Time:year.N	Time:year.N	The time it takes the Earth to complete one revolution of the Sun (between 365.24 and 365.26 days depending on the point of reference).	1226	14-73	Jahr::N	3278
 Time:yesterday.ADV	Time:yesterday.?	The day before today.	1174	14-49	gestern::ADV	3265
+Tool:spoon.N	Food:spoon.N	An implement for eating or serving, a small bowl with a long straight handle.	1378	5-37	Löffel::N	1230

```

Comment:
I would rather put this to the Food category. I think that there's no need for own Tool category after all.

Reply:
Perfect. I agree tool shouldn't be a lexical field.